cmd/kubenav/aws.go

```diff
@@ -55,6 +56,12 @@ type AWSSSOAccount struct {
 // AWSGetClusters returns all clusters which can be accessed with the given
 // credentials.
 func AWSGetClusters(accessKeyID, secretKey, region, sessionToken, roleArn string) (string, error) {
+	defer func() {
+		if r := recover(); r != nil {
+			log.Println("panic: %#v", r)
+		}
+	}()
```

Comment:
I'd recommend doing this so you return the error to the caller as well as logging it:
```suggestion
func AWSGetClusters(accessKeyID, secretKey, region, sessionToken, roleArn string) (_ string, err error) {
	defer func() {
		if r := recover(); r != nil {
			log.Println("panic: %#v", r)
			err = fmt.ErrorF("panic: %w", r)
		}
	}()
````

Reply:
Makes sense and I changed it [here](https://github.com/kubenav/kubenav/pull/801), thank you 🙂